workers/main.go

```
@@ -23,91 +22,71 @@ func getProcessor(task string) (types.TaskProcessor, error) {
+			sentry.CaptureException(err)
+			http.Error(w, "Failed to process task", http.StatusInternalServerError)
+			return
+		}
 
-	select {
-	case <-stop:
-		log.Println("Shutting down gracefully...")
-	case <-forever:
-	}
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte("processed"))
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)